bentoml/_internal/server/grpc_app.py

```diff
@@ -128,21 +132,32 @@ def interceptors(self) -> list[aio.ServerInterceptor]:
             AsyncOpenTelemetryServerInterceptor as AsyncOtelInterceptor,
         )
 
-        # TODO: prometheus interceptors.
-        interceptors: list[t.Type[aio.ServerInterceptor]] = [
-            GenericHeadersServerInterceptor,
-            AsyncOtelInterceptor,
-        ]
+        interceptors: list[aio.ServerInterceptor] = []
+
+        if self.enable_metrics:
+            from .grpc.interceptors.prometheus import PrometheusServerInterceptor
+
+            prometheus_interceptor = PrometheusServerInterceptor(
+                bento_service=self.bento_service
+            )
+            interceptors.append(prometheus_interceptor)  # type: ignore
+
+        interceptors.extend(
+            [
+                GenericHeadersServerInterceptor(),
```

Comment:
This should be at L135

Reply:
Shouldn't prometheus interceptor be the first item on that list?

Reply:
GenericHeaders setups content-type and some others grpc headers